### ISSUE TITLE
Loss function to use

### ISSUE THREAD

**Issue by xuhaoteoh** [KaroDievas/car-sound-classification-with-keras#6]
Hello, 

Do you have any idea which loss function shall I use if I have 4 folders for 4 different car brands? Can I use the same binary_crossentropy function or shall I change it to Multi-Class Cross-Entropy?

I appreciate much if you reply.

With regards,
Teoh

**Comment by KaroDievas** [comment]
Hello,

Thanks for ticket.
binary_crossentropy should be used only if you have two cars. In example let's say you have "Audi" and "Other" so which means you want to find only "Audi" in all other cars.
In your case you should use multi-class cross-entropy as you mentioned.
